apps/music/js/app.js

```diff
@@ -44,18 +44,6 @@ if (navigator.mozSetMessageHandler) {
   navigator.mozSetMessageHandler('activity', activity => onActivity(activity));
 }
 
-var unknownArtist;
-var unknownTitle;
```

Comment:
The localization resources are downloaded and parsed once, and then the only cost of `formatValues` is getting the strings from the cache in a Promise.